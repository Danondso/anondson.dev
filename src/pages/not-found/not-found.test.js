/* eslint-disable max-len */
import React from 'react';
import { useStaticQuery } from 'gatsby';
import renderer from 'react-test-renderer';
import NotFound from './not-found';

describe('Profile Image Snapshot test', () => {
  useStaticQuery.mockReturnValueOnce({
    site: {
      siteMetadata: {
        description: 'A Site Description For A Test',
      },
    },
  });
  it('renders correctly', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
