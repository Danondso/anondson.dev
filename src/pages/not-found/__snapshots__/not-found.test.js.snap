// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Image Snapshot test renders correctly 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <h1>
      404
    </h1>
    <a
      href="/"
    >
      back
    </a>
  </div>
</div>
`;
